WalletWasabi/Extensions/TimeSpanExtensions.cs

```diff
@@ -2,20 +2,28 @@
 using System.Linq;
 using WalletWasabi.Crypto.Randomness;
 
+#pragma warning disable IDE0130 // Namespace does not match folder structure
```

Comment:
If it is not necessary, I would really try not to do this. Wasabi org implemented this code, so it should be under its name. It just asks for backfiring (i.e. conflict).

Reply:
Removed it,